slack-api-model/src/main/java/com/slack/api/model/event/PongEvent.java

```diff
@@ -0,0 +1,21 @@
+package com.slack.api.model.event;
+
+import lombok.Data;
+
+import java.time.Instant;
+
+/**
+ * The pong event is sent in response to a 'ping' message previously sent. The id and
+ * other fields will match that of the ping message.
+ * <p>
+ * https://api.slack.com/rtm
+ */
+@Data
+public class PongEvent implements Event
+{
+	public static final String TYPE_NAME = "pong";
+	private final String type = TYPE_NAME;
+
+	private final Long reply_to;
```

Comment:
Can you define this field as `private Long replyTo` (camelCase + no need to have `final` for this) instead? Lombok generates `Long getReplyTo()` and `void setReplyTo(Long)` for you.

Regarding the conversion from snake_cased JSON keys, Gson in this project automatically translates snake_case to camelCase. https://github.com/slackapi/java-slack-sdk/blob/v1.0.7/slack-api-client/src/main/java/com/slack/api/rtm/RTMEventsDispatcherImpl.java#L53

Reply:
Thanks - good to know!